source/documentation/other-topics/modsecurity.md

```diff
@@ -0,0 +1,108 @@
+### ModSecurity - Web Application Firewall
+
+#### Introduction
+
+[ModSecurity](http://www.modsecurity.org/about.html) is an open source, cross platform web application firewall (WAF) developed by Trustwave's SpiderLabs. It has a robust event-based programming language which provides protection from a range of attacks against web applications and allows for HTTP traffic monitoring and logging. 
```

Comment:
I think it would be helpful to have a sentence or two describing what a WAF is, and why you might want to have one in front of your web application.

Reply:
added what is a waf section